app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
+    def __init__(self, message: str) -> None:
```

Comment:
You raise all exceptions at cafe.py. You don't have to rewrite the same thing in this file, it's enough to write "pass" for each class here.

Reply:
Agree with Yelizaveta